llm-models/code_generation/codellama/codellama-13b/01_load_inference.py

```diff
@@ -0,0 +1,501 @@
+# Databricks notebook source
+# MAGIC %md
+# MAGIC # Code Llama 13B Inference on Databricks
+# MAGIC
+# MAGIC [Code Llama](https://huggingface.co/codellama) is a collection of pretrained and fine-tuned generative text models ranging in scale from 7 billion to 34 billion parameters. It is trained with 2T tokens and supports context length window upto 10K tokens. The model is designed for general code synthesis and understanding.
+# MAGIC
```

Comment:
Can we also add that the notebook contains example for the 3 flavors of the codellama model, and we suggest using the latter 2 for inference?